components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -288,7 +288,7 @@ be lost if SCREAM_HACK_XML is not enabled.
       <atm_procs_list>(shoc,cldFraction,spa,p3)</atm_procs_list>
       <atm_procs_list COMPSET=".*SCREAM.*noAero">(shoc,cldFraction,p3)</atm_procs_list>
       <number_of_subcycles hgrid="ne4np4">24</number_of_subcycles>
-      <number_of_subcycles hgrid="ne30np4">6</number_of_subcycles>
+      <number_of_subcycles hgrid="ne30np4">12</number_of_subcycles>
       <number_of_subcycles hgrid="ne120np4">3</number_of_subcycles>
```

Comment:
We also need to change the ne120 value from 3 to 6 since default full-model dt for ne120 is 15 min so 3 substeps = 5 min, which is too long. We also have evidence of ne120 crashes from this problem as well (from Elynn).

Reply:
Yes, I can also push a commit to do that, but I don't want to hold this up as currently, ne30 is broken on master and has been for a long time.  Just change above 
`<number_of_subcycles hgrid="ne120np4">3</number_of_subcycles>`
to
`<number_of_subcycles hgrid="ne120np4">6</number_of_subcycles>`
?

Reply:
Yup, perfect. It looks like you already made the change. Thanks! ne120 was just as broken as ne30, so it is good to fix both.